### PR TITLE
Add Vonage swag opportunity

### LINKED ISSUE #683: Vonage Hacktoberfest
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : Vonage

Q. Where have you seen the announcement (URL)?  
A. : https://developer.nexmo.com/hacktoberfest

Q. What kind of swag (clothing, stickers, ...)?  
A. : clothing, coupons

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by lazav94**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Fixes #683

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -593,6 +593,15 @@
     "dateAdded": "2019-10-04T16:13:00.000Z",
     "tags": ["hacktoberfest", "stickers", "accessories", "expired"]
   },
+  {
+    "name": "Vonage",
+    "difficulty": "medium",
+    "description": "First, sign up on the Hacktoberfest site. To qualify for the official limited edition Hacktoberfest shirt, you must register and make four pull requests between October 1-31. Pull requests can be to any public repo on GitHub, not just the ones with issues labeled Hacktoberfest. If a maintainer reports your pull request as spam or spammy behavior, or not in line with the project’s code of conduct, you will be ineligible to participate.",
```
Comment (aslafy-z) [inline data.json:599]: Please make it a bit more light, and precise what repos we're talking about.

Comment (aslafy-z) [reply data.json:599]: Also precise the kind of swag, from the "What swag can I earn from Vonage?" section in their FAQ.

File: data.json [hunk 2]
```diff
@@ -593,6 +593,15 @@
...
+    "reference": "https://developer.nexmo.com/hacktoberfest",
+    "image": "https://res.cloudinary.com/dsg3ucw55/image/upload/v1599671200/Logo_nstaby.svg",
+    "dateAdded": "2020-10-01T10:15:00.000Z",
+    "tags": ["hacktoberfest", "stickers", "clothing"]
```
Comment (aslafy-z) [inline data.json:603]:
```suggestion
    "tags": ["hacktoberfest", "stickers", "clothing", "credit/coupon/gift-card"]
```
@vikaspotluri123 wdyt?

**Comment by aslafy-z** [comment]
Vonage is the new Nexmo, should we replace the old entry?

File: data.json (hunk 1 above)
Comment (lazav94) [reply data.json]: Thank you for your comments. The description is updated

File: data.json (hunk 2 above)
Comment (lazav94) [reply data.json]: The tags are updated. I removed the stickers and re-added `credit/coupon/gift-card`.

File: data.json [hunk 3]
```diff
@@ -593,6 +593,15 @@
...
     "tags": ["hacktoberfest", "stickers", "accessories", "expired"]
   },
+  {
+    "name": "Vonage",
+    "difficulty": "medium",
+    "description": "Make a pull request to issues tagged with \"Hacktoberfest\" label and in one of Vonage repos. Check project repos here: https://github.com/Vonage/. After you create PR submit form and get $5 gift card or socks",
+    "reference": "https://developer.nexmo.com/hacktoberfest",
+    "image": "https://res.cloudinary.com/dsg3ucw55/image/upload/v1599671200/Logo_nstaby.svg",
+    "dateAdded": "2020-10-01T10:15:00.000Z",
+    "tags": ["hacktoberfest", "clothing", "credit/coupon/gift-card"]
```
Comment (aslafy-z) [inline data.json:603]:
```
    "tags": ["hacktoberfest", "clothing", "credit/coupon/gift-card"]
```
@vikaspotluri123 Which tag should we use for gift-card from your opinion?

Comment (vikaspotluri123) [reply data.json:603]: Since it's for Open Collective, maybe `donation`?

**Review by aslafy-z — APPROVED** [review]

File: data.json [hunk 4]
```diff
@@ -593,6 +593,15 @@
     "dateAdded": "2019-10-04T16:13:00.000Z",
     "tags": ["hacktoberfest", "stickers", "accessories", "expired"]
   },
+  {
+    "name": "Vonage",
+    "difficulty": "medium",
+    "description": "Make a pull request to issues tagged with \"Hacktoberfest\" label and in one of Vonage repos. Check project repos here: https://github.com/Vonage/. After you create PR submit form and get $5 gift card or socks",
```
Comment (vikaspotluri123) [inline data.json:599]:
```suggestion
    "description": "Make a pull request to a <a href="https://github.com/Vonage">Vonage repo</a> referencing an issue labeled \"Hacktoberfest\", fill out a form, and get $5 gift card or socks",
```

**Review by vikaspotluri123 — APPROVED** [review]

**Comment by aslafy-z** [comment]
Thank you @lazav94!

**Pull request merged**